frontend/src/action-creator.async.js

```diff
@@ -1167,6 +1167,7 @@ export const getAdvancedSearchResult = (
       if (searchFacets.content_types) queryParameterList.push(`content_types=${searchFacets.content_types}`)
       if (searchFacets.file_extensions) queryParameterList.push(`file_extensions=${searchFacets.file_extensions}`)
       if (searchFacets.author__public_names) queryParameterList.push(`author__public_names=${searchFacets.author__public_names}`)
+      if (searchFacets.tags) queryParameterList.push(`tags=${searchFacets.tags}`)
```

Comment:
is searchFacets.tags an array? are the elements url encoded? Does this work with a tag containing a comma?

Reply:
I just added proper array support.